porting/examples/dummy/Makefile

```diff
@@ -0,0 +1,33 @@
+# Configure NimBLE variables
```

Comment:
Can this app share the name with its npl?
Either call the app "template" or rename the npl it uses "dummy".
Is there a way to make an app completely generic, and compile it against your npl of choice?

Reply:
Dummy app and template NPL are not really related. The purpose of this app is only to verify that porting layer builds. Once we have NPL that can be build on Linux we can create simple app using that NPL.

There's no way to make application completely generic since we do not make abstraction of complete OS. NPL only abstracts calls which are required to run NimBLE on other OS.